repoaudit/setup.py

```diff
@@ -5,7 +5,7 @@
 
 setup(
     name="repoaudit",
-    python_requires=">=3.8",
+    python_requires=">=3.6",
```

Comment:
I'm curious about this change especially since Python 3.6 is EOL?

Reply:
I honestly made this change since I have a dev vm running Ubuntu 18.04 and that comes preinstalled with python 3.6. I'll change it back to 3.8.